gffquant/db/annotation_db.py

```
@@ -212,6 +254,29 @@ def get_features(self, category_id):
     def query_category(self, category_id):
         return self.db.categories.get(int(category_id))
 
+    def get_categories(self):
+        yield from self.db.categories.values()
+
+    def get_features(self, category=None):
```

Ruff (0.8.0)

Redefinition of unused `get_features` from line 244

(F811)